packages/dropdowns/README.md

```diff
@@ -0,0 +1,107 @@
+# @zendeskgarden/react-dropdowns [![npm version](https://img.shields.io/npm/v/@zendeskgarden/react-dropdowns.svg?style=flat-square)](https://www.npmjs.com/package/@zendeskgarden/react-dropdowns)
+
+This package includes components relating to dropdowns in the
+[Garden Design System](https://zendeskgarden.github.io/).
+
+## Installation
+
+```sh
+npm install @zendeskgarden/react-dropdowns
+
+# Peer Dependencies - Also Required
+npm install react react-dom prop-types styled-components@3.4.10 @zendeskgarden/react-theming
+```
+
+## Requirements
+
+This package uses several modern React features like [Context](https://reactjs.org/docs/context.html)
+and [Hooks](https://reactjs.org/docs/hooks-intro.html) to provide a more consistent developer experience.
+
+These features require the following minimum versions of the dependencies:
+
+- `react@^16.8.0`
+- `react-dom@^16.8.0`
+- `styled-components@^3.4.0`
+  - `v4` will become available once all other packages are migrated
+
+## Basic Example
+
+```jsx static
+/**
+ * Include dropdowns styling *once* at the root of your application
+ */
+import '@zendeskgarden/react-dropdowns/dist/styles.css';
+
+import { ThemeProvider } from '@zendeskgarden/react-theming';
+import { Dropdown, Menu, Item, Trigger } from '@zendeskgarden/react-dropdowns';
+
+/**
+ * Place a `ThemeProvider` at the root of your React application
+ */
+<ThemeProvider>
+  <Dropdown onSelect={value => console.log(`Selected: ${value}`)}>
+    <Trigger>
+      <button>This triggers a menu</button>
+    </Trigger>
+    <Menu placement="end" arrow>
+      <Item value="option-1">Option 1</Item>
+      <Item value="option-2">Option 2</Item>
+      <Item value="option-3">Option 3</Item>
+    </Menu>
+  </Dropdown>
+</ThemeProvider>;
+```
+
+For all components within the `react-dropdowns` package, the menu layouts and
+implementations are interchangeable.
+
+Whether you're making a `Select`, `Autocomplete`, or a traditional `Menu` the `<Menu>` implementation
+will adapted to it's consumer.
+
+## Usage
+
+### Overview
+
+The `react-dropdowns` package abstracts the common concepts of `Menus`, `Selects`, and `Autocompletes`
+into a common API. This includes consistent visuals, common keyboard interaction, and a fully accessible
+experience for sighted and non-sighted users.
+
+The customizations available within this can be broken into two groups: _placement / positioning_
+and _dropdown state_
+
+### Placement / Positioning
+
+Internally, the `<Dropdown>` component uses [PopperJS](https://popper.js.org/popper-documentation.html)
+for its positioning calculations.
+
+The `<Menu>` component accepts all customizations regarding placement, boundaries, overflows,
+etc. via the `popperModifiers` prop.
```

Comment:
Is it consistent to name this `popperModifiers` while using `downshiftProps` below?

Reply:
`downshiftProps` actually get spread onto the `Downshift` element. This is just a 1-to-1 prop match with PopperJS since framework agnostic.